vignettes/opencage.Rmd

```diff
@@ -59,7 +59,7 @@ Whatever method you choose, keep your API key secret. OpenCage also features [be
 ### Rate limit
 
 A rate limit is used to control the rate of requests sent, so legitimate requests do not lead to an unintended Denial of Service attack.
-The rate limit allowed by the API depends on the OpenCage plan you have and ranges from 1 request/sec for the "Free Trial" plan to 15 requests/sec for the "Medium" or "Large" plans.
+The rate limit allowed by the API depends on the OpenCage plan you have and ranges from 1 request/sec for the "Free Trial" plan to 40 requests/sec for "Large" plan.
```

Comment:
how do you keep track of pricing changes?

Reply:
I don't (systematically). I just follow the twitter account and browse the site every once in a while for changes.